### ISSUE TITLE
Login fail with social account Facebook 

### ISSUE THREAD

**Issue by dirceudn** [ReachFive/identity-android-sdk#62]
Hi,

When I try a login with a social account Facebook I have an error like this:
```

ReachFiveError(message=User cancel, code=null, exception=null, data=null) 
and 
ReachFiveError(message=No provider found for this requestCode: 4, code=null, exception=null, data=null)
```

**Comment by roxanemace** [comment]
Hi @dirceudn, this should be fixed with the [`v5.3.2`](https://github.com/ReachFive/identity-android-sdk/releases/tag/v5.3.2) release :)

**Comment by dirceudn** [comment]
Hi,
@roxanemace Thanks
